src/globalInstance.ts

```diff
@@ -10,14 +10,11 @@ export function getGlobalInstance(): Monitor {
     if (!instance) {
         instance = new Monitor({
             serviceName: '',
+            plugins: [],
         });
     }
 
     return instance;
 }
 
 export const monitored: Monitor['monitored'] = (...args) => getGlobalInstance().monitored(...args);
-export const getStatsdClient: Monitor['getStatsdClient'] = (...args) => getGlobalInstance().getStatsdClient(...args);
-export const increment: Monitor['increment'] = (...args) => getGlobalInstance().increment(...args);
-export const gauge: Monitor['gauge'] = (...args) => getGlobalInstance().gauge(...args);
-export const timing: Monitor['timing'] = (...args) => getGlobalInstance().timing(...args);
```

Comment:
I think it's a major change because today you can mock these functions or use them directly in your code.
But I'm not sure.

Reply:
This entire change is very breaking so this will surely be a semver-major change.

I would however work on a migration guide. Since monitored used to be centered around `statsd` and now it's more plugin-based, we need to provide a migration regarding these methods.

It'd probably mean exposing the stated plugin instance and using it to call the specific monitoring methods (`increment`, `gauge`, `timing`).